bind_exporter.go

```diff
@@ -214,33 +209,25 @@ func (e *Exporter) Collect(ch chan<- prometheus.Metric) {
 
 	status = 1
 
-	root := Isc{}
-	if err := xml.Unmarshal([]byte(body), &root); err != nil {
+	stats, err := unmarshal(body)
+	if err != nil {
 		log.Error("Failed to unmarshal XML response: ", err)
 		return
 	}
 
-	stats := root.Bind.Statistics
-	for _, s := range stats.Server.QueriesIn.Rdtype {
```

Comment:
Yeah looks like your removing a bunch of stats. Would probably be safe to just keep both sets